### PR TITLE
Add a dummy C++ application

### PR DESCRIPTION
**Pull request by GordanM**
The dummy application currently prints a simple "Hello" message. A CMake
file for the application has been added, as well as the systemd .service
file. There are no installation directives for the .service file as it is
provided only as an example.

Signed-off-by: Gordan Markuš <gordan.markus@pelagicore.com>

### CODE REVIEW

**Review by zeenix — APPROVED** [review]
Looks fine to me. :)

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Ignore CMake cache files
+CMakeCache.txt
+**/CMakeFiles/
+**/*.cmake
+**/Makefile
+
+# Ignore shadow build folder
```
Comment (breakreturn) [inline .gitignore]: It's a "directory" man :)

The desktop domain has destroyed today's youth.

**Review by breakreturn — APPROVED** [review]
Had a super nitpicky comment. Feel free to take care of it or not, I don't think it should block a merge.

File: .gitignore (hunk 1 above)
Comment (GordanM) [reply .gitignore]: @breakreturn you are right, got some of those comments from the Qt guys as well :). Thanks.

Comment (breakreturn) [reply .gitignore]: ;)

**Pull request merged**